NodeTypes.Schema.json

```diff
@@ -136,6 +136,14 @@
                                     ],
                                     "minimum": 2,
                                     "default": 6
+                                },
+                                "maxRows": {
+                                    "type": [
```

Comment:
Can you add a `description` with the version information, like we have for some other fields already?
And adjust the example for the test in `examples/NodeTypes.Editors.yaml` :)

Reply:
Done